docs/en/latest/discovery/kubernetes.md

```diff
@@ -351,3 +351,55 @@ A: Assume your [_ServiceAccount_](https://kubernetes.io/docs/tasks/configure-pod
  ```shell
  kubectl -n apisix get secret kubernetes-discovery-token-c64cv -o jsonpath={.data.token} | base64 -d
  ```
+
+## Debugging API
+
+It also offers control api for debugging.
+
+### Memory Dump API
+
```

Comment:
Add a descriptive sentence something like "to query/list the nodes discoverd by kubernetes discovery, you can query the `/v1/discovery/kubernetes/dump` control API endpoint like so:"

Reply:
ok